src/style.css

```diff
@@ -0,0 +1,25 @@
+* {
+  font-size: 13px;
+  font-family: Arial;
+}
```

Comment:
don't use `*` selector for font, it's mostly used for styles such as `box-sizing`